app/controllers/plaid_items_controller.rb

```diff
@@ -5,6 +5,7 @@ def create
     Current.family.plaid_items.create_from_public_token(
       plaid_item_params[:public_token],
       item_name: item_name,
+      region: plaid_item_params[:region]
```

Comment:
The `create_from_public_token` method on `PlaidItem` will need to be updated to accept this param